packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -612,13 +615,13 @@ class ActionDispatcher with Diagnosticable {
   }
 }
 
-/// A widget that establishes an [ActionDispatcher] and a map of [Intent] to
```

Comment:
A newbie can implement shortcuts and invoke actions manually without ever using `ActionDispatcher`. I've deprioritized it as it enables advanced scenarios.

Reply:
Makes sense.